src/Views/EnterDisplayName.js

```diff
@@ -12,28 +13,68 @@ export default function EnterDisplayName(props) {
         API.join(name, props.match.params.roomID)
         dispatch({ type: 'update-name', displayName: name })
     }
-    return (
-        <div className="create-page">
-            <div className="p-4 m-4">
-                <h4 className="display-name">
-                    Display Name <span className="required">*</span>
-                </h4>
-                <br />
-                <form onSubmit={handleJoinClick}>
-                    <input
-                        type="text"
-                        id="displayName"
-                        placeholder="Enter Display Name"
-                        class="form-control"
-                        onChange={(e) => setName(e.target.value)}
-                        required
-                    />{' '}
+    function handleRejoinClick(e) {
+        e.preventDefault()
+        API.join(name, props.match.params.roomID)
+        dispatch({ type: 'update-name', displayName: name})
+        dispatch({ type: 'remove-from-kicked', userId: contextState.userId})
+    }
+
+    if(contextState.kicked.includes(contextState.userId) === true)
```

Comment:
This is superfluous. We don't need an entire array if the entire point is to check if a single user has been kicked. I'd rather a boolean value here in the application state than an array.

Reply:
We are also branching for no reason. The code paths are nearly the same.

Reply:
If we are sending a boolean with the userId that is being kicked, how would I reference that specific userId?

Reply:
I agree that the boolean would be easier

Reply:
Would it be with the draft case?